openpype/hosts/maya/plugins/publish/collect_pointcache.py

```diff
@@ -43,17 +43,29 @@ def process(self, instance):
             instance.remove(proxy_set)
             instance.data["setMembers"].remove(proxy_set)
 
-        # Collect user defined attributes.
-        if not instance.data.get("includeUserDefinedAttributes", False):
-            return
+        # Backwards compatibility for attributes.
+        backwards_mapping = {
```

Comment:
Maybe we could do this directly in the `create/create_point_cache.py`

Reply:
You mean the backwards compatibility?
This needs to happen in the collector cause the it'll pick up on legacy instances whereas the creator will just be making a new one.